gear/legacy_identifier_transfer/src/python/legacy_identifier_transfer_app/main.py

```diff
@@ -0,0 +1,212 @@
+"""Defines legacy_identifier_transfer."""
+
+from datetime import datetime
+import logging
+from typing import Dict, Mapping
+
+from pydantic import ValidationError
+
+from enrollment.enrollment_project import EnrollmentProject
+from enrollment.enrollment_transfer import EnrollmentRecord
+from gear_execution.gear_execution import GearExecutionError
+from identifiers.model import CenterIdentifiers, IdentifierObject
+from outputs.errors import ErrorWriter, ListErrorWriter, identifier_error, legacy_naccid_error, unexpected_value_error
+
+log = logging.getLogger(__name__)
+
+
+class LegacyEnrollmentBatch:
```

Comment:
I don't think it is necessary to batch these if it is easier to do otherwise. Batching in the identifier-provisioning gear is due to the input being a file with a batch of form records.